rootfs/scheduler/__init__.py

```diff
@@ -190,7 +43,7 @@
 
 SECRET_TEMPLATE = """\
 kind: Secret
-apiVersion: $version
+apiVersion: v1
```

Comment:
why do we want to hard code the version..wouldn't it be simpler during an upgrade using the existing class variable